packages/core/helper-plugin/package.json

```diff
@@ -58,8 +58,7 @@
     "react-intl": "5.20.2",
     "react-router": "^5.2.0",
     "react-router-dom": "5.2.0",
-    "styled-components": "^5.2.3",
-    "whatwg-fetch": "^3.6.2"
```

Comment:
This lib is still imported in the helper-plugin/lib/utils/request can you remove the import as well?

Reply:
sorry it took this long. Just done this change